frontend/src/components/order/order.module.css

```diff
@@ -0,0 +1,22 @@
+.container {
```

Comment:
You may want to rename this to be more specific to the type of container since, eventually, there will likely be many types of containers.

Reply:
It's a module file--class names aren't global. But yeah, it's still general